services/121-service/test/payment/do-payment-fsp-nedbank.test.ts

```diff
@@ -0,0 +1,517 @@
+import { HttpStatus } from '@nestjs/common';
+
+import { ExportType } from '@121-service/src/metrics/enum/export-type.enum';
+import { NedbankVoucherStatus } from '@121-service/src/payments/fsp-integration/nedbank/enums/nedbank-voucher-status.enum';
+import { TransactionStatusEnum } from '@121-service/src/payments/transactions/enums/transaction-status.enum';
+import { ImportRegistrationsDto } from '@121-service/src/registration/dto/bulk-import.dto';
+import { SeedScript } from '@121-service/src/scripts/enum/seed-script.enum';
+import { adminOwnerDto } from '@121-service/test/fixtures/user-owner';
+import {
+  doPayment,
+  exportList,
+  getTransactions,
+  retryPayment,
+  waitForPaymentTransactionsToComplete,
+} from '@121-service/test/helpers/program.helper';
+import {
+  seedIncludedRegistrations,
+  seedPaidRegistrations,
+  updateRegistration,
+} from '@121-service/test/helpers/registration.helper';
+import {
+  getAccessToken,
+  resetDB,
+  runCronjobUpdateNedbankVoucherStatus,
+} from '@121-service/test/helpers/utility.helper';
+import { registrationNedbank } from '@121-service/test/registrations/pagination/pagination-data';
+
+const programId = 1;
+const payment = 1;
+const amount = 200;
+
+enum NedbankMockNumber {
+  failDebitorAccountIncorrect = '27000000001',
+  failTimoutSimulate = '27000000002',
+}
+
+enum NebankGetOrderMockReference {
+  orderNotFound = 'mock-order-not-found',
+  mock = 'mock',
+  phoneNumberIncorrect = 'mock-phone-number-incorrect',
+}
+
+describe('Do payment to PA(s)', () => {
```

Comment:
```suggestion
describe('Do payment', () => {
```
We do not use the term PA anymore.